reference/shaders-hlsl/vert/basic.vert

```diff
@@ -0,0 +1,47 @@
+struct _UBO
+{
+    float4x4 uMVP;
+};
+
+cbuffer UBO
+{
+    _UBO _16;
+};
+uniform float4 gl_HalfPixel;
```

Comment:
This is not HLSL, how will this work? Is this gl_HalfPixel there to handle the infamous DX9 window space offset?